ts/models/message.ts

```diff
@@ -567,6 +585,16 @@ export class MessageModel extends Backbone.Model<MessageAttributes> {
     });
   }
 
+  public getPropsForReacts(): ReactionList | null {
+    const reacts = this.get('reacts') || null;
+
+    if (!reacts) {
+      return null;
+    }
+
+    return reacts;
```

Comment:
this is just

`return this.get('reacts') || null`

Reply:
Good point, think this was code I wrote in the beginning